src/Schema/Events/ImpressionTracking.ts

```diff
@@ -259,3 +259,28 @@ export interface EditProfileModalViewed {
   inquiry_id: string
   platform: Platform
 }
+
+/**
+ * User sees the banner in the app
+ *
+ * This schema describes events sent to Segment from [[BannerViewed]].
+ *
+ *  @example
+ *  ```
+ *  {
+ *    action: "bannerViewed",
+ *    context_screen: "home",
+ *    context_module: "paymentFailed",
```

Comment:
I added this `paymentFailed` as a `context_module` which in combination with `bannerViewed` and `context_screen` will allow to indicate `paymentFailed banner on home screen`. Think it's descriptive enough. If we potentially put this same banner on other screens we'll be able to call the same with different `context_screen`. That makes sense to me.